rotkehlchen/assets/asset.py

```diff
@@ -563,6 +563,12 @@ def to_dict(self) -> dict[str, Any]:
             'underlying_tokens': underlying_tokens,
         }
 
+    def decimals_or_default(self) -> int:
```

Comment:
rename to `get_decimals()`

Or even perhaps make a getter the python way ... though probanbly this is fine